.gitmodules

```
@@ -1,3 +1,4 @@
 [submodule "docs/source/pymc-examples"]
 	path = docs/source/pymc-examples
 	url = git@github.com:pymc-devs/pymc-examples.git
```

I presume this doesn't work for you because you don't have ssh keys set up. Perhaps we can change this line to
```suggestion
	url = https://github.com/pymc-devs/pymc-examples.git
```

---

Ah, I see @MarcoGorelli. Thanks!

Did that. Following, I did a `git submodule sync`, based [on this article](https://stackoverflow.com/questions/11410017/git-submodule-update-over-https).
_____
Process now looked like:
0. Edit the `.gitmodules` file
1. `git submodule sync`
2. `git submodule update --remote`
3.  `git checkout main` (on the submodule)
4. Went to parent module, and committed changes